src/components/Header/index.jsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import scrollTo from 'gatsby-plugin-smoothscroll';
+
+import Logo from '../Logo/';
+import Menu from '../Menu/';
+import SocialMedia from '../SocialMedia';
+import ScrollDown from '../ScrollDown/';
+import Title from '../../components/common/Title/';
+import Description from '../../components/common/Description/';
+
+import { HeaderWrapper, ContainerHeader, HeaderLogo, ContentHeader, BtnWrapper, BtnHeader, ScrollWrapper } from './styled';
+
```

Comment:
Please create a constant in order to get rid of inline javascript functions, something like this 

```suggestion
const scrollToTarget = () => (id) => scrollTo(id)
```
and in your component, you can call it in this way:

```

          <BtnHeader onClick={scrollToTarget('#Header')}>
            start a project
          </BtnHeader>
```

Reply:
This doesn't need to be done on this PR thought

Reply:
Done.

Reply:
Done.